tests/cpp/test_multidevice_host_ir_integration.cpp

```diff
@@ -0,0 +1,67 @@
+// clang-format off
+/*
+* SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+* All rights reserved.
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+// clang-format on
+#include <fusion.h>
+#include <host_ir/container.h>
+#include <host_ir/executor.h>
+#include <ir/all_nodes.h>
+#include <ops/all_ops.h>
+#include <tests/cpp/multidevice.h>
+
+namespace nvfuser {
+
+namespace hir {
+
+TEST_F(MultiDeviceTest, LaunchKernel) {
```

Comment:
```suggestion
using HostIrIntegrationTest = NVFuserTest;

TEST_F(HostIrIntegrationTest, LaunchKernel) {
```
I believe this test doesn't need the MultiDeviceTest fixture.